libp2p/transports/wstransport.nim

```diff
@@ -269,9 +271,9 @@ method accept*(
     try:
       await one(self.acceptFuts)
     except ValueError:
-      raiseAssert("already checked with if")
+      raiseAssert("already checked with if: " & getCurrentExceptionMsg())
```

Comment:
```suggestion
      raiseAssert("already checked with if")
```
No need to add the exception here. This is an error that wont happen and just added to make the compiler happy